requirements/requirements.in

```
@@ -3,7 +3,7 @@
 # if you are add a new feature which requires dependencies they should be in a separate requirements_<feature>.in file
 #
 cryptography
-Django>=4.2.21,<6.0
+Django>=4.2.26,<6.0
```

Can we add a comment as to why? like:
```suggestion
Django>=4.2.26,<6.0    # Updating for CVE-2025-64459
```